docs/en/observability/aws-serverless-troubleshooting.asciidoc

```diff
@@ -19,6 +13,15 @@ You can view the status of deployment actions and get additional information on
 
 NOTE: For example, if you don't increase the visibility timeout for an SQS queue as described in <<aws-serverless-forwarder-inputs-s3>>, you will see a `CREATE_FAILED`**Status** for the event, and the **Status reason** provides additional detail.
 
+[[aws-serverless-troubleshooting-vpc-prerequisites]]
+== Prerequisites when attached to a VPC
+If the Elastic Serverless Forwarder is attached to a VPC, you need to https://docs.aws.amazon.com/vpc/latest/privatelink/create-interface-endpoint.html[create VPC Endpoints] for S3 and SQS, and for *every* service you define as an input for the forwarder. S3 and SQS VPC Endpoints are always required for reading the `config.yaml` uploaded to S3 and managing the _Continuing queue_ and the _Replay queue_, regardless of the <<aws-serverless-forwarder-inputs>> used.
+
+== Preventing unexpected costs
+It is important to monitor the Elastic Serverless Forwarder Lambda function for timeouts to prevent unexpected costs. You can use the https://docs.elastic.co/en/integrations/aws/lambda[AWS Lambda integration] for this. If the timeouts are constant, you should throttle the Lambda function to stop its execution before proceeding with any troubleshooting steps. In most cases, constant timeouts will cause the records and messages from the event triggers to go back to their sources and trigger the function again, which will cause further timeouts and force a loop that will incure unexpected high costs.
+
+// is it clear how you would throttle the Lambda function? should we detail and number these steps?

```

Comment:
this is something specific to AWS: we could link AWS documentation, I'm just worried that if AWS will change the link or similar our docs will be out of sync